sdk/ml/azure-ai-ml/azure/ai/ml/dsl/_do_while.py

```diff
@@ -38,20 +38,19 @@ def pipeline_with_do_while_node():
 
     :param body: Pipeline job for the do-while loop body.
```

Comment:
Do we support other types as while body now? Or just subgraphs?

Reply:
Currently a simple command component is acceptable, but the created pipeline job is wired. Need to investigate.

Reply:
Command component is also supported.

Reply:
Added command in docstring for body.